client/css/widgets/label.css

```diff
@@ -1,10 +1,10 @@
-.label {
+.widget.label {
```

Comment:
Is there a reason for increasing the specificty here.

Reply:
As I continue adding new UI stuff these more specific declarations help keep cross-contamination at bay. Since I had it open, I thought I would adjust it.